README.md

```diff
@@ -372,6 +374,119 @@ const keyMap = {
 };
 ```
 
+#### Setting dynamic hotkeys
+
+`react-hotkeys` has basic support for setting dynamic hotkeys - i.e. letting the user set their own keyboard shortcuts. In your app, you can set up the necessary UI for [viewing the current keyboard shortcuts](#displaying-a-list-of-available-hot-keys), and opting to change them. You can then use the `recordKeyCombination` function to capture the keys the user wishes to use.
```

Comment:
⬜️📚Change start of second sentence to `Once you have set up the necessary UI ...`